lib/Form/Type/Language/LanguageType.php

```diff
@@ -0,0 +1,37 @@
+<?php
+/**
+ * This file is part of the eZ RepositoryForms package.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ * @version //autogentag//
+ */
+namespace EzSystems\RepositoryForms\Form\Type\Language;
+
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+class LanguageType extends AbstractType
+{
+    public function buildForm(FormBuilderInterface $builder, array $options)
+    {
+        $builder
+            ->add('name', 'text')
+            ->add('identifier', 'text')
+            ->add('save', 'submit', ['label' => 'language.form.save']);
+    }
+
+    public function getName()
+    {
+        return 'ezrepoforms_language_edit';
+    }
+
+    public function configureOptions(OptionsResolver $resolver)
+    {
+        $resolver->setDefaults([
+            'data_class' => '\eZ\Publish\API\Repository\Values\Content\LanguageStruct',
```

Comment:
`LanguageStruct` doesn't exist... As far as I can see, you can set this to `LanguageCreateStruct`